test/v1/ProxyNegativeTests.js

```diff
@@ -96,13 +96,6 @@ async function run_tests(newToken, _accounts) {
     await checkVariables([token], [customVars]);
   });
 
-  it("nut007 should fail to call proxy function with non-adminAccount", async () => {
```

Comment:
This test is removed because the `ifAdmin` modifier for `admin()` and `implementation()` are removed. There wasn't really a point since those addresses were visible regardless. Besides, those were marked as view functions, but if a non-admin invokes those functions, the modifier could cause a delegate call to the functions with the same name in the implementation contract which may not be read-only (i.e. not view). New versions of Solidity compiler correctly catches this and errors out. The implementation contract should also never contain meaningful `admin()` and `implementation()` overrides that relies on the modifier to be callable anyway.